src/java.base/share/conf/security/java.security

```diff
@@ -85,10 +85,11 @@ security.provider.tbd=Apple
 #endif
 security.provider.tbd=SunPKCS11
 
-#if defined linux-x86 || defined linux-ppc || defined linux-s390
+#if defined aix-ppc || defined linux-ppc || defined linux-s390 || defined linux-x86 || defined windows
 #
 # Java Restricted Security Mode
 #
+#if defined linux-ppc || defined linux-s390 || defined linux-x86
```

Comment:
With this addition should there not also be an added `#endif`?

Reply:
`MakeJavaSecurity` doesn't appear to support nested conditions, so I think that new `#endif` belongs before line 92 so conditions are not nested here. Otherwise I think it's misleading (the similarity with the C preprocessor might lead readers to think nesting is supported leading to a misunderstanding of meaning here).

Reply:
Yes, the `#endif` is needed here. I updated the codes and right now, rebuilding it on Jenkins.